### PR TITLE
Added Cask to install 8x8 Work Application

### PR DESCRIPTION
**Pull request by cduranleau**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @cduranleau.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

File: Casks/8x8-work.rb [hunk 1]
```diff
@@ -7,5 +7,10 @@
   desc "Communications application with voice, video, chat, and web conferencing"
   homepage "https://www.8x8.com/products/apps"
 
+  livecheck do
+    url "https://support.8x8.com/cloud-phone-service/voice/work-desktop/download-8x8-work-for-desktop#Download_for_Mac"
+    regex(/work-dmg-v(\d+(?:.\d+)*)\.dmg/i)
```
Comment (bevanjkay) [inline Casks/8x8-work.rb:12]:
```suggestion
regex(/work[._-]dmg[._-]v(\d+(?:.\d+)*)\.dmg/i)
```
@miccal

Comment (miccal) [reply Casks/8x8-work.rb]: Thanks @bevanjkay.

**Comment by cduranleau** [comment]
> Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

Yes of course!! I THINK I found all of the files created as a part of the application.

**Comment by miccal** [comment]
Thank you @cduranleau.

**Comment by cduranleau** [comment]
Thank you @miccal for the help!!! :) Have a great weekend!!!

**Pull request merged**